webclient/app/src/app/commons/form/UpdateFormStyles.js

```diff
@@ -0,0 +1,15 @@
+import {makeStyles} from "@mui/styles";
+
+const UpdateFormStyles = makeStyles(theme => ({
+    textField: {
+        marginLeft: theme.spacing.unit,
```

Comment:
Ich würde persönlich hier kein Margin einbauen

Reply:
das ist keine sinnvolle Begründung. Das UpdateForm würde ich grundsätzlich erst einmal so lassen. Kann in einem separaten PR angepasst werden - nicht hier.

Reply:
Klar - Die Begründung hatte ich ja mal gesagt, als wir telefoniert hatten. Hätte ich vlt mal ergänzen sollen.
Hast Recht, das ist hier contained aber ich sehe noch nicht so wirklich den Zweck hierfür. Man kann dann auch ein <Container> Element verwenden, was sich dann responsive anpasst

Reply:
https://dev.azure.com/starwit/lj-projectbuilder/_workitems/edit/248